kustomize/kustomization.yaml

```diff
@@ -19,5 +19,6 @@ resources:
   - base
 
 components:
-  - components/cymbal-branding
-  - components/google-cloud-operations
+  # - components/cymbal-branding
+  # - components/google-cloud-operations
+  - components/memorystore
```

Comment:
**Question:**
Was this change intentional?
I think, by default, `kustomize/kustomization.yaml` should not use `components/memorystore`.
Feel free to ignore — not a deal breaker.